mappings/net/minecraft/client/gui/menu/BlastFurnaceScreen.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS czx net/minecraft/client/gui/menu/BlastFurnaceScreen
+	FIELD l BG_TEX Lqr;
```

Comment:
Since this PR is touching all GUI files already, could be worth finding some consistency in how these textures are named. Is it known how Mojang names these?

Reply:
The button texture is names something like BUTTON_LOCATION or something. I can double check